XamarinCommunityToolkitSample/Models/Language.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Xamarin.CommunityToolkit.Sample.Models
+{
+    public class Language
```

Comment:
Looks like we will use this into business logic, so can we transform this into a `readonly struct`?

Reply:
Done!